2013-04-24  Jean-Charles Delay  <delay@adacore.com>

	(gettext-tools/gnulib-lib/stpncpy.c): Workaround builtin definition
	issue on Mountain Lion.

--- gettext-tools/gnulib-lib/stpncpy.c.orig	2010-05-24 11:42:47.000000000 +0200
+++ gettext-tools/gnulib-lib/stpncpy.c	2013-04-03 18:05:13.000000000 +0200
@@ -25,7 +25,7 @@
 #include <string.h>
 
 #ifndef weak_alias
-# define __stpncpy stpncpy
+//# define __stpncpy stpncpy
 #endif
 
 /* Copy no more than N bytes of SRC to DST, returning a pointer past the
